client/blocks/inline-help/contextual-help.js

```diff
@@ -1485,8 +1485,8 @@ export function getContextResults( section ) {
 
 	// make sure editorially to show at most one tour and one video at once
 	// `first` is a safe-guard in case that fails
-	const video = first( get( videosForSection, section ) );
-	const tour = first( get( toursForSection, section ) );
+	const video = get( videosForSection, section )?.shift();
```

Comment:
Doesn't `Array.shift` mutate the array? That'd be different to `.first()` which gets the first element, but let's the array in place. Is that something we should consider? Or does it matter for those use cases at all?

Reply:
```> arr = [1,2,3]
[ 1, 2, 3 ]
> arr.shift()
1
> arr
[ 2, 3 ]```

It does mutate the array, I don't think it's a safe way to do this.

Reply:
I agree it does mutate the array, but as long as we don't use the array anymore, it should be fine, no?

Reply:
`videosForSection[ section ]?.[ 0 ]` looks like a perfect replacement to me 🙂

Reply:
`get( videosForSection, section )` retrieves the reference, there's no guarantee that the array isn't being used somewhere else.

Reply:
> but as long as we don't use the array anymore

The `videosForSection` array is a global variable. Each call to `getContextResult` (it's called in a Redux selector) will eat one item from the array, like Pacman 🙂

Reply:
I'm glad y'all are paying attention to the detail 👏 😉  Going to fix this tomorrow.